mne/channels/montage.py

```diff
@@ -304,6 +305,49 @@ def _get_dig_names(self):
 
         return dig_names
 
+    def get_positions(self):
+        """Get all channel and fiducial positions.
+
+        Returns
+        -------
+        positions : dict
+            A dictionary of the positions for channels (``ch_pos``),
+            coordinate frame (``coord_frame``), nasion (``nasion``),
+            left preauricular point (``lpa``),
+            right preauricular point (``rpa``),
+            Head Shape Polhemus (``hsp``), and
+            Head Position Indicator(``hpi``).
+            E.g.::
+
+                {
+                    'ch_pos': {'EEG061': [0, 0, 0]},
+                    'nasion': [0, 0, 1],
+                    'lpa': [0, 1, 0],
+                    'rpa': [1, 0, 0],
+                    'hsp': [np.nan, np.nan, np.nan],
```

Comment:
This is a strange result, I would have expected `[]` or `None` here instead of `[np.nan, np.nan, np.nan]`

Reply:
Yes you're right. They end up getting returned as `None` from `_get_data_as_dict_from_dig()` if they are not set.